src/backends/sqlite3/statement.cpp

```diff
@@ -19,12 +19,6 @@
 #pragma warning(disable:4355)
 #endif
 
-// This is used instead of tolower() just to avoid warnings about int to char
-// casts inside MSVS std::transform() implementation.
-char toLowerCh(char c) {
-    return static_cast<char>( std::tolower(c) );
-}
-
```

Comment:
Removing this and inlining it looks a completely gratuitous change, was there really some reason for doing it?